android/app/src/main/AndroidManifest.xml

```diff
@@ -22,9 +22,14 @@
             android:theme="@style/SplashTheme"
             android:configChanges="orientation|screenSize|keyboardHidden">
             <intent-filter>
-                <action android:name="android.intent.action.MAIN" />
-
+                <action android:name="android.intent.action.VIEW"/>
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
                 <category android:name="android.intent.category.LAUNCHER" />
+
+                <data
+                    android:scheme="http"
+                    android:host="2017.fossasia.org" />
```

Comment:
Don't hardcode this here.
Instead add the url to config.json and fetch it from there.